nix/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  dbus,
+}:
+rustPlatform.buildRustPackage (_finalAttrs: {
```

Comment:
If Nix is to be added, I think I'd prefer for it to use [crate2nix](https://nix-community.github.io/crate2nix).